en/docs/modules/misc/VideoChip.md

```diff
@@ -88,6 +88,10 @@ end
 
 <img src="../../../assets/docs/VideoChip/SpriteFont.png" width="200" align="right">
 
+⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
+### DrawCustomSprite(position `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
```

Comment:
These lines should be rearranged to follow the Method Name -> Method Details convention.
```suggestion
### DrawCustomSprite(position `vec2`, spriteSheet `SpriteSheet`, spriteOffset `vec2`, spriteSize `vec2`, tintColor `color`, backgroundColor `color`)
⚠️ **This feature is new, as of Retro Gadgets 0.1.3, and not fully tested yet**
```